### ISSUE TITLE
Change password length HTTP error in loopback 4

### ISSUE THREAD

**Issue by JuliaGmP** [strongloop/loopback-next#3939]
I am using loopback 4 authentication and I have this credential schema. In my register method I am getting this http error response when giving a password shorter than 8 characters. I'm trying to avoid error message in the response that the password should be shorter than 8 characters, instead respond with an http error response without that details.

I would like to know where this error definition is and change it to a new one with less information.

I tried something like this

```
if (credential.password.length < 8) {
      throw new HttpErrors.NotFound;
} 
```
in the register method but it did not work. I could delete the minLength limit 8 in the schema but I want that error response in that case.

```
export const CredentialsSchema = {
  type: 'object',
  required: ['email', 'password'],
  properties: {
    email: {
      type: 'string',
      format: 'email',
   },
   password: {
     type: 'string',
     minLength: 8,
   },
  },
};


async register(
@requestBody() credential: CredentialPassword,
): Promise<AccessToken> {
return this.userRegistration(credential);
}


{
"error": {
"statusCode": 422,
"name": "UnprocessableEntityError",
"message": "The request body is invalid. See error object `details` property for more info.",
"code": "VALIDATION_FAILED",
"details": [
  {
    "path": ".password",
    "code": "minLength",
    "message": "should NOT be shorter than 8 characters",
    "info": {
      "limit": 8
    }
  }]
 }
}
```
Thank's for your help !

**Comment by dhmlau** [comment]
@emonddr, could you please take a look? Thanks.

**Comment by achrinza** [comment]
This is probably fixed by #4623

**Comment by achrinza** [comment]
Closing due to inactivity.

@JuliaGmP - Feel free to open a new issue if the problem persists.
